zend_abstract_interface/symbols/lookup.c

```diff
@@ -67,6 +67,8 @@ static inline void *zai_symbol_lookup_table(HashTable *table, zai_string_view ke
     } else {
         return result;
     }
+#else
```

Comment:
unused under 7, but this is a high frequency function and I didn't want to litter the callsites with ifdefs